sdk/notificationhubs/notification-hubs/review/notification-hubs.api.md

```diff
@@ -56,28 +56,42 @@ export interface AdmNotification extends JsonNotification {
 }
 
 // @public
-export interface AdmRegistrationDescription extends RegistrationDescriptionCommon {
+export interface AdmRegistrationChannel {
     admRegistrationId: string;
-    type: "Adm";
+    kind: "adm";
 }
 
 // @public
-export interface AdmTemplateRegistrationDescription extends Omit<AdmRegistrationDescription, "type">, TemplateRegistrationDescription {
-    type: "AdmTemplate";
+export interface AdmRegistrationDescription extends AdmRegistrationDescriptionCommon {
```

Comment:
similarly, I wonder if we could collapse some of these types, even at the cost of repeating ourselves in places. The interface compatibility should still work out and it's a bit hard to follow the relationships right now

Reply:
Right, the Omit<> isn't working for the docs, but perhaps in the future when we fix that, it'll be easier.  It doesn't break anything right now to add this hierarchy though